cypress/component/integration/Clipboard.cy.tsx

```diff
@@ -65,35 +65,6 @@ describe('Component | Integration | Clipboard', function () {
           'formatted content',
         );
       });
-
-      it('should fire the copy clipboard function', () => {
```

Comment:
revert commit that remove the clipboard dep and add it to optimize dep of vite config